xrpl4j-core/src/test/java/org/xrpl/xrpl4j/model/transactions/json/NfTokenCreateOfferJsonTests.java

```diff
@@ -55,10 +56,39 @@ public void testMinimalNfTokenCreateOfferJson() throws JsonProcessingException,
       "    \"Sequence\": 12,\n" +
       "    \"Amount\": \"2000\",\n" +
       "    \"NFTokenID\": \"000B013A95F14B0044F78A264E41713C64B5F89242540EE208C3098E00000D65\",\n" +
-      "    \"SigningPubKey\" : \"02356E89059A75438887F9FEE2056A2890DB82A68353BE9C0C0C8F89C0018B37FC\",\n" +
-      "    \"Flags\": 2147483648\n" +
```

Comment:
All these `Flags` removals make me nervous.

Reply:
I removed all the `Flags` for ledger objects/transactions that do not have specific flags. All of these classes will now have empty flags because the `flags` field is `Derived`. However, for all transactions/ledger objects with specific flags, I included a variant of the JSON tests with and without flags.